docs/scalars.md

```diff
@@ -88,14 +88,14 @@ class User {
   registrationDate: Date;
 }
 ```
-
-If you use `ts-node`, be aware you must execute it with the `--type-check` flag due to a [Date reflection bug](https://github.com/TypeStrong/ts-node/issues/511).
+Note:
```

Comment:
Basically it has been fixed, so I would ask you to delete this note by the way 😉

Reply:
OK, deleted note.